tests/v2rdm_casscf/v2rdm4/input.dat

```diff
@@ -40,6 +40,7 @@ set {
   basis cc-pvdz
   scf_type df
   d_convergence 1e-8
+  ints_tolerance 0.0
```

Comment:
:(

Reply:
yes, but it's 1 of 8 tests for a code where convergence is the big problem (maxiter 10k), so I'm not too downcast.